engine/dumps/bank03.asm

```diff
@@ -2285,29 +2285,29 @@ Datae236:
 	dw Texte23e
 
 Texte23e:
-	db "#の　ようすをみる@"
+	db "#の　ようすをみる@" ; (lit "look at Pokemon")
 
 Texte248:
-	db "#を　つれていく@"
+	db "#を　つれていく@" ; "Withdrawal (Pokemon)"
 
 Texte251:
-	db "#を　あずける@"
+	db "#を　あずける@" ; "Deposit (Pokemon)"
 
 Texte259:
-	db "#を　にがす@"
+	db "#を　にがす@" ; "Release (Pokemon)"
 
 Texte260:
-	db "ボックスを　かえる@"
+	db "ボックスを　かえる@" ; "Change Box"
 
 Texte26a:
 	db "さようなら@"
 
 Tablee270:
 	dw Functione5c5
 	dw Functione31b
-	dw Functione2a6
-	dw Functione37b
-	dw Functione3c3
+	dw Functione2a6 ; Bill's PC > Deposit Pokemon menu item
+	dw Functione37b ; Bill's PC > Release Pokemon menu item
+	dw Functione3c3 ; Bill's PC > Box Change menu item
```

Comment:
Please rename labels rather than commenting them. Comments are fine to translate text strings, but you shouldn't comment labels. This applies to everything else in this PR.